app/model/sdl/Abstract/AppModel.js

```diff
@@ -665,7 +665,49 @@ SDL.ABSAppModel = Em.Object.extend(
      * @description Called after receiving CreateWindow request 
      */
     createWindow: function(windowParam) {
-      windowParam.content = {"templateConfiguration" : this.defaultTemplateConfiguration};
+      var content = {};
+      if (windowParam.duplicateUpdatesFromWindowID === 0) {
+        var showStringsArray = [];
+        SDL.SDLController.model.appInfo.field1 ? showStringsArray.push({
+          "fieldName": "mainField1",
+          "fieldText": SDL.SDLController.model.appInfo.field1
+        }) : null ;
+
+        SDL.SDLController.model.appInfo.field2 ? showStringsArray.push({
+          "fieldName": "mainField2",
+          "fieldText": SDL.SDLController.model.appInfo.field2
+        }) : null ;
+
+        content["showStrings"] = showStringsArray;    
+
+        var softButtonsArray = this.get("softButtons");
+        if (softButtonsArray.length > 4) {
+          softButtonsArray.length = 4;
+        }
+        
+        content["softButtons"] = softButtonsArray;
```

Comment:
```suggestion
        content["softButtons"] = this.get("softButtons").slice(0,4);
```
optional: could save a few lines, i think it looks better